src/base/NavItemType.php

```diff
@@ -25,14 +25,20 @@ abstract class NavItemType extends ActiveRecord implements NavItemTypeInterface
      */
     abstract public function getContent();
 
+    private $_navItem;
+    
     /**
      * Get the corresponding nav item type for this type object
      *
      * @return NavItem An active record type.
      */
     public function getNavItem()
     {
-        return $this->hasOne(NavItem::class, ['nav_item_type_id' => 'id'])->where(['nav_item_type' => static::getNummericType()]);
+        if ($this->_navItem === null) {
```

Comment:
this is not need. using ->navItem does the same

Reply:
Thanks for your advice!

Reply:
I've reverted that commit on `NavItemType.php`.